src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/TelemetryService.kt

```diff
@@ -52,6 +53,20 @@ class TelemetryService(private val telemetryClient: TelemetryClient) {
     private val log = LoggerFactory.getLogger(this::class.java)
   }
 
+  fun trackEventWithIds(
+    eventType: TelemetryEventType,
+    person: Person,
+    elementMap: Map<EventKeys, String?> = emptyMap(),
+  ) {
+    val identifierMap = mapOf(
+      EventKeys.SOURCE_SYSTEM to person.sourceSystemType.name,
+      EventKeys.DEFENDANT_ID to person.defendantId,
```

Comment:
I'm not sure this is allowed
One bit of PII is OK e.g. CRN, but something like this which would allow a picture of an individual to be put together might not be. Do we really need this?

Reply:
Yeah, it only will log out one of them as the rest will be null due to the source system ID 👍